src/screens/internal/CompareSnapshots/CompareSnapshots.tsx

```diff
@@ -418,7 +418,7 @@ function useProjectionsSet(
           ),
         );
       } else if (locations === Locations.TOP_COUNTIES_BY_POPULATION) {
-        const topCounties = topCountiesByPopulation(COUNTIES_LIMIT);
+        const topCounties = regions.topCountiesByPopulation(COUNTIES_LIMIT);
```

Comment:
Can we delete the version of this method in common/locations now?

Reply:
oo probably, i'll check